client/src/ui/BitbakeWorkspace.ts

```diff
@@ -37,6 +37,14 @@ export class BitbakeWorkspace {
     this.onChange.emit('recipeDropped', chosenRecipe)
   }
 
+  dropAllActiveRecipes (): void {
+    this.activeRecipes = []
+    if (this.memento !== undefined) {
+      void this.saveBitbakeWorkspace(this.memento)
```

Comment:
I'd suggest to await this properly. Otherwise it is not clear what will complete first between the saving of the new state and the reactions to `recipeDropped`. I suspect this is that sort of things that makes our tests fail once in a while.

Reply:
I haven't seen the test fail randomly in weeks. The errors we had were not related to the memento at all. It's only read at the extension activation as well so it's probably not much trouble.
The only race condition I can imagine is if two quick elements hapen at the same time (probably not possible through the current bitbakeCommands), then we can save the workspace out of order which won't have any major effect.

I'm going to add a commit to await properly as a good practice but I don't expect it to fix anything.